docs/src/userguide/extension_types.rst

```diff
@@ -39,14 +39,14 @@ interface to them.
 
 .. _readonly:
 
-Attributes
-============
+Static Attributes
+=================
 
 Attributes of an extension type are stored directly in the object's C struct.
 The set of attributes is fixed at compile time; you can't add attributes to an
 extension type instance at run time simply by assigning to them, as you could
 with a Python class instance. (You can subclass the extension type in Python
-and add attributes to instances of the subclass, however.)
+and add attributes to instances of the subclass, see :ref:`dynamic_attributes`.)
```

Comment:
I'd add a note here already that subclassing is not the only way:
"... Python class instance. However, you can explicitly enable support for dynamically assigned attributes, or subclass the extension type with a normal Python class, which then supports arbitrary attribute assignments.  See :ref:`dynamic_attributes`."